### ISSUE TITLE
Add possibility of copying projects and tasks

### ISSUE THREAD

**Issue by Daniel451** [opencv/cvat#5438]
At the moment, we can only move tasks from one project to another but not copy tasks or projects. However, this would come in very handy if we would like to have different Train/Val/Test configurations of the same dataset and manipulate annotations. 

Use case example:
Let's consider you want to test how accurate segmentation annotations have to be made for your dataset. Instead of re-creating annotations entirely it would be much nicer to copy an already existing project (or individual tasks for that matter if you just want to run a small test), maybe assigning more samples to "Test" in this project and just making the segmentation masks a little more precise by going over the existing annotations and then simply adding some points to the polygon shapes.

Currently, we have to download the project, edit the data manually, re-upload as a new project, and then modify or, even worse, start from scratch.

A simple copy option for projects & tasks would come in very handy in such cases.

**Comment by nmanovic** [comment]
@Daniel451 ,probably you want to "version" your project and make branches for experiments. Also you don't want to download your data. Do all manipulation on the server side. Am I right?

**Comment by FawadAbbas12** [comment]
is there any possibility to implement this feature ?

> @Daniel451 ,probably you want to "version" your project and make branches for experiments. Also you don't want to download your data. Do all manipulation on the server side. Am I right?

**Comment by Daniel451** [comment]
> probably you want to "version" your project and make branches for experiments. Also you don't want to download your data. Do all manipulation on the server side. Am I right?

@nmanovic yes, exactly. A direct way of versioning would also be nice but especially copying for creating branches for experiments.
